README.md

```diff
@@ -68,6 +94,13 @@ python
 >>> import waveorder
 ```
 
+(Optional) Install napari, clone the repository, and run an example script:
+```sh
+pip install napari[all]
```

Comment:
We can potentially put napari and jupyter in optional dependencies e.g. `pip install "waveorder[examples]".

Reply:
Great idea!

I just updated `pyproject.toml` so that `pip install "waveorder[examples]"` installs napari and jupyter, tested a fresh install, and updated both `README`s.